config/initializers/sentry.rb

```diff
@@ -1,9 +1,7 @@
 Raven.configure do |config|
   config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
 
-  if ENV['OPENSHIFT_BUILD_COMMIT']
-    config.release = ENV['OPENSHIFT_BUILD_COMMIT']
-  end
+  config.release = Rails.application.class.versions(Rails.root.join('VERSION')).first.chomp
```

Comment:
Would it make sense to keep using the BUILD on integration and only read VERSION for production releases?

Reply:
IMHO it would not make sense. Theoretically, one could go an find the exact build with this and try to reproduce this locally. I have never done this and cannot imagine doing it. The version however gives me a useful approximation to decide how fixed an older bug might be and where to check in the source.

For production, every build should have a different tag. So the build-commit is mostly the same, just less readable.

Reply:
If you find the build-commit  useful as additional tag, we can add that back in. PullRequests are welcome :-)